ioservice/io_foms.c

```diff
@@ -1797,8 +1797,9 @@ static int io_launch(struct m0_fom *fom)
 		m0_fom_callback_arm(fom, &stio->si_wait, &stio_desc->siod_fcb);
 		m0_mutex_unlock(&stio->si_mutex);
 
-		M0_LOG(M0_DEBUG, "launch fom: %p, start_time %lu, req_count: %lx, "
-		       "count: %lx, submitted: %lx, expect: %lx",
+		M0_LOG(M0_DEBUG, "launch fom: %p, start_time %"PRIi64", "
+		       "req_count: %"PRIx64", count: %"PRIx64", "
+		       "submitted: %"PRIx64", expect: %"PRIx64,
```

Comment:
s
/"req_count: %"PRIx64", count: %"PRIx64", "
/"req_count: 0x%"PRIx64", count: 0x%"PRIx64", "

Reply:
s
/"submitted: %"PRIx64", expect: %"PRIx64,
/"submitted: 0x%"PRIx64", expect: 0x%"PRIx64,

Reply:
That's fine, original output was in hexadecimal without 0x. I don't want to break existing formats.